sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -207,5 +203,5 @@ def shutdown_plugin(self):
     rospy.init_node("sr_data_visualizer")
     app = QApplication(sys.argv)
     data_visualiser_gui = SrDataVisualizer(None)
-    data_visualiser_gui._widget.show()
+    data_visualiser_gui._widget.show()  # pylint: disable=W0212
```

Comment:
Weird to have a private attribute called in here... But I guess it's not on you :P

Reply:
Yeah, I would change if I knew why :(

Reply:
I think it's just something that we missed during previous PR :(